src/macros.rs

```diff
@@ -1,10 +1,305 @@
+//! Extra macro documentation.
+
 #[doc(hidden)]
 pub use fizyr_rpc_macros::interface as interface_impl;
 
 #[macro_export]
 /// Define an RPC interface.
+///
+/// This macro generates a module with a `Client` and `Server` struct,
+/// and some more helper types.
```

Comment:
Maybe briefly add what these helper types are intended for? Or is it so small that it isn't noteworthy?

Reply:
No, it's more that there are so many, and they're subdivided into multiple modules =]